_data/data-sponsor.yml

```diff
@@ -0,0 +1,236 @@
+      - name: 红叶故事
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/c_pad,h_224,w_224/v1636691899/zgzg-io-website/b5dcJAY_dd9ixh.png
+        description: 红叶故事，可以视频聊天的北美婚恋交友App。来红叶朋友圈晒生活，浏览周围单身小伙伴的动态，发送照片，语音和表情。
+
+      - name: george-xie
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1656789735/xie_dbxfoz.png
+        description: George Xie 谢晓光 - 加州地产经纪人，Keller Williams TOP 1% Agent，CIPS。凭借着独特的私人银行和贷款金融背景，为客户提供全方面的房产置业资产配置咨询。超高的谈判沟通能力帮助客户拿到最好的价格。
+
+      - name: LingoAce
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1656783718/lingo_yviz8v.png
+        description: 暑期宅家也要好好学习呀！LingoAce海外中文线上免费领取1对1试听中文课！听完的小朋友可以领取价值$20的amazon礼卡哟！加微信BayAreaKidFun了解领取福利详情！每周还会解锁湾区最新亲子活动噢!
+
+      - name: Coco Tan
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1661481301/zgzg-io-website/logo4_yvmiyq.jpg
+        description: Coco Tan从业地产18年有余, 一直排名前1%。在全美140万名地产经纪人中, Coco名列由《华尔街日报》评选的前74名。2021年还未结束, 她已卖出了140多套房屋, 几乎每套都成功地打破了湾区的价格记录。她曾连续两年当选华美地产协会 (CAREA) 主席。
+
+      - name: Steve Chen
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1659917448/zgzg-io-website/IMG_6399_ovz1dp.jpg
+        description: Steve Chen 富国银行房屋贷款经理. Steve 作为资深房屋贷款顾问及销售经理，服务在湾区享有极好口碑。他的专长包括 Relocation Loan，涉及 RSU 收入及海外资产的贷款。
+
+      - name: Daniel Xi
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1659917449/zgzg-io-website/20YH1F-DC_SF_DP_-_Custom_Logo_-_Daniel_Xi_-_SOE-blk-FINAL_1_rbacdp.jpg
+        description: Top 1% Realtor，经验丰富，谈判技巧高，服务质量一流，无论买房卖房，都很擅长。2021年个人业绩全美排名第157名，交易量1.7亿美元。
+
+      - name: Ariel Chen
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1660113834/zgzg-io-website/20YH1F-DC_SF_DP_-_Custom_Logo_-_Ariel_Chen_-_SOE-blk-FINAL_qr1v6_envmot.jpg
+        description: 深耕地产行业多年，始终奉行将客户的利益放在第一位。买卖房屋，尤其擅长换房，给每个家庭提供定制化换房策略。
+
+      - name: Wukong
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1662160921/zgzg-io-website/2022%E6%98%A5%E6%99%9A%E6%8B%8D%E6%91%84%E8%8A%B1%E7%B5%AE/Logo2_g1vp7v.png
+        description: 风靡全球的WuKong在线课堂来到硅谷啦！赶快添加微信公众号【悟空中文】【悟空国际数学】0元领取试听课和学习资料大礼包！3-18岁孩子们的专属福利🎁
+
+    - level-name: musical_instrument
+      level-heading: 乐器赞助
+      elements-div-style-class: col-6 col-lg-3 mb-1 gold-sponsor
+      sponsor-elements:  
+      - name: Enya智能乐器
+        image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1661795574/zgzg-io-website/2022%E6%98%A5%E6%99%9A%E6%8B%8D%E6%91%84%E8%8A%B1%E7%B5%AE/IMG_6517_zfwmhh.jpg
+        description: 恩雅音乐是一家集乐器、智能音频、音乐文化、为一体的创新型科技音乐公司。超过十年的沉淀让恩雅更好的服务着全球音乐爱好者们。
+
+    - level-name: venue
+      level-heading: 场地赞助
+      elements-div-style-id: last-row-partners
+      elements-div-style-class: col-6 col-lg-3 mb-1 food-sponsor
```

Comment:
Question: Why is it needed to add these 2 properties?

We preferably want to separate **styles** from **data**. Preferably, this YAML file should only contain **data**, and it's the HTML templates' job to **style** the content correctly.

If there's a good reason to add these 2 properties here, maybe you can add a comment explaining the necessity?

Reply:
Added comments to explain the reasons.